docs/api-reference/widgets/fullscreen-widget.md

```diff
@@ -0,0 +1,68 @@
+# FullscreenWidget
+
+## Variables
+
+### Size
+
+| Name | Type | Default |
+| ---- | ---- | ------- |
+| `--button-size` | [Dimension](https://developer.mozilla.org/en-US/docs/Web/CSS/dimension) | `28px` |
+| `--button-border-radius` | [Dimension](https://developer.mozilla.org/en-US/docs/Web/CSS/dimension) | `12px` |
+| `--wdgt-margin` | [Dimension](https://developer.mozilla.org/en-US/docs/Web/CSS/dimension) | `12px` |
```

Comment:
Users have a variety of ways to control the scope of their CSS variable customizations, and these are good additions for the docs.

To scope to the fullscreen widget users can set the `style` prop in the JS widget - that uses a CS in-line style, so it'll cascade down only for that instance. This would mean two fullscreen buttons can be uniquely styled if desired.

To target all fullscreen widgets variables can be set CSS in the `.deckgl-fullscreen` selector.

To target all widgets set variables in `.deckgl-widget`.

Reply:
As for prefix's, I'd like them to be semantic names that aren't prescriptive of look.

So `widget` prefix should target the widget as a whole, `button` targets whatever is functionally apart of a button, etc. 

An good, non-prescriptive, name for a CSS `backdrop-filter` property would be `button-backdrop-filter`. `button-backdrop-blur` wouldn't be good since `blur` prescribes a look.